CONTRIBUTING.md

```diff
@@ -10,15 +10,32 @@ In the first place, thank you for your interest in contributing! 🙏
 
 ## Development
```

Comment:
I am still not satisfied with using Docker inside of NPM scripts. Let's discuss it outside of Github on Google Meet and write resolution on that later.

Reply:
On the Google Meet we did not manage to come to an agreement. I said I would outline the options:

1. Start everything through `npm start` <== current state
  * + comfy for developers
  * + unified workflow through all of our projects
  * - `npm` starts unrelated technology (Docker)
2. Start `npm` and Docker independently
  * + no unexpected behavior when starting `npm`
  * + different workflow from our other projects
  * - developers have to run two commands in two consoles. The docker starting command is very long and must be copied from `README.md`
 3. Start everything by a bash script `start.sh`
  * + no unexpected behavior when starting `npm`
  * + comfy for developers
  * - different workflow from our other projects
  
  
I'm not suggesting using make or any other task runner as conceptually that is option 3) with the drawback of introducing new technology that provides featured that we don't really need.

There was a sugestion by @bedrich-schindler to start Docker by `npm` via a dedicated command (e.g. `npm run docker:start`). I dislike that, because command organization logic is about code and not about technology (`lib`, `docs`). We already have a dedicated command `docs:start` that start docker and that is what @bedrich-schindler dislikes because of unexpected behavior when starting `npm`.

Honestly, I would just leave it as is because I can think of no good solution and current state is the most comfortable for day-to-day development. I don't really feel strong about it though, so please do let me know what you think.

Reply:
Chceme verzi 3)

Reply:
Cool, thank you for outlining the options 👍🏻. Number 3️⃣ seems to be my preferred one.

Reply:
done